transform/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "transform-step-extension",
+  "version": "0.1.0",
+  "private": true,
+  "port": 9001,
+  "devDependencies": {
+    "@babel/core": "7.15.5",
+    "@babel/preset-react": "^7.18.6",
+    "@babel/preset-typescript": "7.15.0",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^14.4.3",
+    "@types/jest": "^29.2.6",
+    "@types/node": "^12.0.0",
+    "@types/react": "^18.0.27",
+    "@types/react-dom": "^18.0.10",
+    "babel-loader": "8.2.2",
+    "css-loader": "^6.2.0",
+    "html-webpack-plugin": "5.3.2",
+    "jest": "^29.3.1",
+    "jest-environment-jsdom": "^29.3.1",
+    "mini-css-extract-plugin": "^2.7.2",
+    "node-polyfill-webpack-plugin": "^2.0.1",
+    "null-loader": "^4.0.1",
+    "serve": "12.0.1",
+    "style-loader": "^3.2.1",
+    "ts-jest": "^29.0.5",
+    "typescript": "4.3.5",
+    "util": "^0.12.5",
+    "webpack": "5.52.1",
+    "webpack-cli": "4.10.0",
+    "webpack-dev-server": "4.2.0",
+    "webpack-merge": "^5.8.0"
+  },
+  "dependencies": {
+    "@apidevtools/swagger-parser": "^10.1.0",
+    "@patternfly/patternfly": "^4.222.4",
+    "@patternfly/react-core": "^4.267.6",
+    "@patternfly/react-styles": "^4.92.3",
+    "buffer": "^6.0.3",
+    "openapi-types": "^12.1.0",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "4.0.3",
+    "stream-http": "^3.2.0",
+    "web-vitals": "^1.0.1"
+  },
+  "scripts": {
+    "start:dev": "webpack-cli serve --https --hot --color --progress --config webpack.dev.js",
+    "start": "webpack-cli serve --hot --color --progress --config webpack.dev.js",
+    "build": "webpack --mode production --config webpack.common.js",
+    "serve": "serve dist -p 3002",
+    "clean": "rm -rf dist",
+    "startbu": "react-scripts start",
```

Comment:
We can probably remove all of these `bu` scripts, they are just backups of the original, leftover